modules/core/src/main/java/org/apache/ignite/internal/processors/cache/GridCacheProcessor.java

```diff
@@ -4568,6 +4568,13 @@ public DynamicCacheDescriptor cacheDescriptor(String name) {
         return cachesInfo.registeredCaches();
     }
 
+    /**
+     * @return Cache ID to Name Mapping
+     */
+    public Map<Integer, String> cacheIDToNameMapping() {
```

Comment:
Should be called cacheIdToNameMapping I guess, based on https://cwiki.apache.org/confluence/display/IGNITE/Coding+Guidelines#CodingGuidelines-Naming